tests/01-auth/01-signUp/02-signUpNegativePassword.spec.js

```diff
@@ -0,0 +1,19 @@
+import { test } from '../../_fixtures/fixtures';
+import { INVALID_CONFIRM_PASSWORD } from '../../../src/ui/constants/authMessages';
+
+test(`Confirmation password is wrong`, async ({
```

Comment:
```suggestion
test(`Assert validation message is shown when Confirmation password is different from Password field`, async ({
```

or smth like this may look better